test/client_test.js

```diff
@@ -115,14 +115,14 @@ suite('api (client)', function() {
       c => c.clientId.substr(CLIENT_ID.length)).sort();
 
     // get all clients
-    assume(gotSuffixes(await helper.auth.listClients())).to.deeply.equal(suffixes);
```

Comment:
Oh dear, I missed this in the last round of review.  This is bad -- it means anything that expects to get a plain list of clients back will fail.

@jonasfj what do you think -- how should we solve this?

Reply:
If we fix tc-tools, and tc-admin + tc-cli, I think we're safe... But we should probably fix those to handle both cases first..

That's just a suggestion, it might also be acceptable to break... However, given that the API is declared stable... Perhaps we should make some noise about this change on a mailing list... Ask if anyone is worried...

Maybe look through logs to see how much it's used...